### DISCUSSION TITLE
How to move PR forward?

### DISCUSSION THREAD

**Discussion by yuri-karelics** [Leaflet/Leaflet#9162] [General]
I've created PR to the repo in September https://github.com/Leaflet/Leaflet/pull/9100. It is November, but PR stuck and I don't understand what is wrong and how to move it forward.
Did I make a mistake in description or there is some other problem with it (hopefully not with me)? Let's start discussion, or just close PR if it somehow doesn't fit existing logic.

**Comment by jonkoops** [comment]
We're all maintaining Leaflet in our spare time, and we unfortunately cannot guarantee that we can give your PR our attention. Please be patient until we are able to get to your work.
